package.json

```diff
@@ -32,9 +32,8 @@
   },
   "devDependencies": {
     "lodash": "^2.4.1",
-    "jstransform": "^6.2.0",
+    "jsx-loader": "0.13.2",
```

Comment:
I'd prefer to keep this change in another PR.

I've been playing a little bit with `babel` and tend to like it.  Would want a little more usage before making the switch.